src/components/Films/FilmsComp.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './FilmsComp.css';
+export default function Films({ films }) {
+  // console.log(films);
+
+  return (
+    <div className='films-container'>
+      {films.map((film) => (
+        <div key={film.id} className='film'>
+          <h3>{film.title}</h3>
+          <img src={`${process.env.PUBLIC_URL}/films/${film.title.replaceAll(' ', '-').toLowerCase()}.jpeg`} />
```

Comment:
Nice job getting the film images to render and replacing the spaces with dashes.